active/online-community.md

```diff
@@ -0,0 +1,112 @@
+# Online Community Working Group
+
+## Scope of responsibilities
+
+This group exists to support the official community platforms and those in elevated roles on those platforms.
+
+The [Django Forum](https://forum.djangoproject.com/) and the [Django Discord](https://discord.gg/xcRH6mN4fa) are core parts of the community and those that provide help and moderate these platforms take significant time from their day to help others.
+We can support them through training and taking action to make their lives easier.
+
+The broad goals of the group are:
+  - Facilitate and centralise discussion between moderators and admins on the various platforms.
+  - Coordinate efforts for training and processes around moderation.
+  - Provide support to improve the platforms to lighten moderation and admin workloads.
+  - Provide training and support to the wider community on how to use the platforms effectively.
+  - Ensure that our online communities are welcoming and safe spaces.
+
+### What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms. Examples of these processes include:
+    - Proposals for adding/removing moderators & other roles and the adding/removing of roles directly.
+    - Training for moderators & other roles as required.
+    - Ensuring moderation is consistent across teams and platforms.
+  - Taking ideas from the community to improve the community spaces.
+
+### Which actions will the WG take back to the Board for votes?
+  - Proposals to trial a new platform.
+  - Proposals to migrate to a new platform.
+  - Proposals to retire an existing platform that is in use.
+  - Budget requests to improve existing platforms.
+
+There are a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the members of group are assigned:
+
+- Administrator Privileges on approved community platforms with the following conditions to create redundancy and protect the community from any single person holding too much power.
+  - A single member may not have Administrator Privileges on all platforms.
+  - Each platform has a minimum of two members with Administrator Privileges.
+
+## Initial membership
+
+- Chair: Andrew Miller
+- Co-Chair: TBD
+- Board Liaison: Tom Carrick
+- CoC Liaison: Dan Ryan
+- SC Liaison?:
```

Comment:
Should either drop this (and the line mentioning it) or have it in place before voting. I'm okay with either.

Reply:
(and if it's dropped, fine to add it later, of course)

Reply:
I'm having difficulty understanding the relation between this working group's activities and the Steering Council. As of right now I don't think a liaison is needed.

Reply:
I agree with @tim-schilling. Having too many Liaisons feels like a thing when what may be needed are some points of how this WG can work with or help with SC vs. a formal reporting structure.